cmd/manager/main.go

```diff
@@ -107,8 +108,19 @@ func main() {
 		os.Exit(1)
 	}
 
+	//Create metrics endpoint and register metrics
+	metricsServer := custommetrics.NewBuilder().WithPort("9090").WithPath("/metrics").
+		WithCollectors(metrics.MetricTotalAWSAccounts).
+		WithCollectors(metrics.MetricTotalAccountCRs).
+		WithCollectors(metrics.MetricTotalAccountCRsUnclaimed).
+		WithCollectors(metrics.MetricTotalAccountCRsClaimed).
+		WithCollectors(metrics.MetricTotalAccountCRsFailed).
```

Comment:
I didn't find the PendingVerificationCount metric in this list of collectors

Reply:
:+1: will add